server/src/OnDeviceComponent.ts

```diff
@@ -245,19 +237,24 @@ export class OnDeviceComponent {
 		this.conditionallyAddDefaultNodeReferenceKey(args);
 
 		const result = await this.sendRequest('deleteNodeReferences', {...args, convertResponseToJsonCompatible: false}, options);
-		return result.body as ODC.ReturnTimeTaken;
+		return result.json as ODC.ReturnTimeTaken;
 	}
 
 	public async disableScreenSaver(args: ODC.DisableScreensaverArgs, options: ODC.RequestOptions = {}) {
 		const result = await this.sendRequest('disableScreenSaver', args, options);
-		return result.body as ODC.ReturnTimeTaken;
+		return result.json as ODC.ReturnTimeTaken;
+	}
+
+	public async focusNodeAtKeyPath(args: ODC.FocusNodeAtKeyPathArgs, options: ODC.RequestOptions = {}) {
```

Comment:
For this name. Is there a focusNode that _doesn't_ require a keypath? If not, then I'm not sure the `AtKeyPath` is actually necessary. `focusNode` takes a keyPath argument. keyPath is simply an implementation detail of the action `focusNode` and doesn't really need to be in the name.

Reply:
Interesting point. If I was going to change this then I likely would change it everywhere. Again I was matching the name on the Roku side. In this case since its my own function I would ok simplifying this side to not include it so `getValue` `setValue` `focusNode` etc

Reply:
updated